objectbox/lib/src/native/bindings/bindings.dart

```diff
@@ -9,41 +9,71 @@ export 'objectbox-c.dart';
 
 // ignore_for_file: public_member_api_docs
 
-ObjectBoxC loadObjectBoxLib() {
+// Tries to use an already loaded objectbox dynamic library. This is the only
+// option for macOS and iOS and should be faster for other platforms as well.
+ObjectBoxC? _tryObjectBoxLibProcess() {
+  // [DynamicLibrary.process()] is not supported on windows, see its docs.
+  if (Platform.isWindows) return null;
+
+  final lib = DynamicLibrary.process();
+  try {
+    final obxc = ObjectBoxC(lib);
+    if (_isSupportedVersion(obxc)) {
+      return obxc;
+    }
+  } catch (_) {}
+  return null;
```

Comment:
My initial idea was to try to load the library from file in case the one in the process is a wrong version... But yeah, after a second thought, let's skip that "gracefulness" until we need it